src/brackets.js

```diff
@@ -142,7 +142,7 @@ define(function (require, exports, module) {
     
     // read URL params
     params.parse();
-    
+
```

Comment:
Could you undo these changes to the whitespace lines? Brackets by default does not trim whitespaces. There's no reason to introduce a change to this. If you have a trim extension, it should probably be disabled for brackets project.